tests/integration/values/values_helm_fluentbit_containerd_multiline_logs.yaml

```diff
@@ -0,0 +1,136 @@
+sumologic:
+  setupEnabled: true
+  accessId: "dummy"
+  accessKey: "dummy"
+  endpoint: http://receiver-mock.receiver-mock:3000/terraform/api/
+  metrics:
+    enabled: false
```

Comment:
As one last thing I'd ask to rebase this PR and remove this as well as the fluentd resource section since that is already covered by https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2041

And then only override what's not set in common yaml.

Reply:
done